File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.c
#include <stdio.h>
int main(void)
{
    int t;
    scanf("%d", &t);
    for(int i=0;i<t;i++)
    {   
        int tst[1000];
        int n, j, avr, cnt=0, sum=0;
        scanf("%d", &n);
        for(j=0;j<n;j++)
        {
            scanf("%d", &tst[j]);
            sum+=tst[j];
        }
        avr=sum/n;
        for(j=0;j<n;j++)
            if(tst[j]>avr)
                cnt++;
        printf("%.3f%% \n", (double)cnt/n*100);
    }
    return 0;
}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
import sys, decimal
input = sys.stdin.readline

context = decimal.getcontext()
context.rounding = decimal.ROUND_HALF_UP

t = int(input())
for i in range(t):
    nums = list(map(int, input().split()))
    avg = sum(nums[1:])/nums[0]
    cnt = 0
    for score in nums[1:]:
        if score > avg:
            cnt += 1
    result = cnt/nums[0]*100
    print(f"{round(result, 3):.3f}%")